src/index.js

```diff
@@ -76,7 +78,8 @@ class ActiveStorageProvider extends React.Component<Props> {
         Accept: 'application/json',
         'Content-Type': 'application/json',
         ...(token ? { Authorization: token } : {}),
-        ...csrfHeader(),
+        ...addCSRFHeader && csrfHeader(),
```

Comment:
Flow doesn’t like when you spread booleans into objects, even though it works.

```suggestion
        ...(addCSRFHeader ? csrfHeader() : {}),
```